db/migrate/20210815214807_add_colunt_name_surname_addres_to_order.rb

```diff
@@ -0,0 +1,7 @@
+class AddColuntNameSurnameAddresToOrder < ActiveRecord::Migration[6.1]
+  def change
+    add_column :orders, :mane_users, :string
+    add_column :orders, :surname_user, :string
```

Comment:
Bad practice to duplicate user`s surname in order. You already have relation with it, no need

Reply:
@VolodymyrZyblikevich